spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestDeleteReachableFilesAction.java

```diff
@@ -329,7 +333,7 @@ public void testEmptyIOThrowsException() {
   }
 
   @Test
-  public void testRemoveFilesActionWhenGarabageCollectionDisabled() {
+  public void testRemoveFilesActionWhenGarbageCollectionDisabled() {
```

Comment:
Wasn't it renamed to `DeleteFilesAction`?

Reply:
yeah. I was mainly focused on fixing `Garbage` spelling.  Didn't see whether whole testcase name is applicable or not

Reply:
many things in this file need to be renamed from `Remove` to `Delete` then

Reply:
I think we should do this rename as well.

Reply:
Oops, I just saw that there are many file names and file contents that use Remove instead of Delete. That is not the scope of this PR to handle. I will stop here for this PR to avoid too many changes and confuse the reviewer.

Reply:
Sure we can do it in a follow up

Reply:
yep. I will handle it in follow up. Thanks